packages/@react-aria/tag/src/useTag.ts

```diff
@@ -80,19 +85,31 @@ export function useTag<T>(props: AriaTagProps<T>, state: ListState<T>, ref: RefO
   let descProps = useDescription(description);
 
   let isFocused = item.key === state.selectionManager.focusedKey;
+  let isNonFocused = state.selectionManager.focusedKey == null;
+  function getTabIndex(
+    isDisabled: boolean,
+    isFocused: boolean,
+    isNonFocused: boolean
+  ) {
+    if (isDisabled) {
+      return -1;
+    }
+    return isFocused || isNonFocused ? 0 : -1;
+  }
+
```

Comment:
```suggestion
  let tabIndex = -1;
  f (!isDisabled && (isFocused || state.selectionManager.focusedKey == null)) {
    tabIndex = 0;
  }
```

We can just derive it as part of render, no need to add an additional function call with arguments we already have.